recipe/meta.yaml

```diff
@@ -21,101 +21,18 @@ requirements:
     - pip
   run:
     - python
-    - langchain-core >=0.1.52,<0.2.0
-    - sqlalchemy >=1.4,<3
+    - langchain <0.4.0,>=0.3.11
+    - langchain-core <0.4.0,>=0.3.24
+    - sqlalchemy <2.0.36,>=1.4
     - requests >=2.0.0,<3.0.0
     - pyyaml >=5.3
     - numpy >=1.0.0,<2.0.0
     - aiohttp >=3.8.3,<4.0.0
     - tenacity >=8.1.0,<9.0.0
     - dataclasses-json >=0.5.7,<0.7
```

Comment:
These should come from [here](https://github.com/langchain-ai/langchain/blob/langchain%3D%3D0.3.12/libs/community/pyproject.toml#L31), right?

Reply:
Updated